src/fileformats/ocd_file_import.h

```diff
@@ -376,8 +382,14 @@ class OcdFileImport : public Importer
 	/// The visibility of layout objects (true if hidden)
 	bool layout_objects_hidden;
 	
+	/// stores the object numbers of hidden layout objects
+	QSet<int> hidden_layout_objects;
+	
 	/// The protection and visibility state of image objects (bitfield: 1 = protected, 2 = hidden)
 	int image_objects_displaymode;
+	
+	/// counting objects during import
+	int object_index;
```

Comment:
I'm not completely sure about the purpose and naming of this member. Can you make its purpose more obvious, please?